src/main/resources/static/styles.css

```diff
@@ -118,4 +118,54 @@
 
 .logoutForm {
     display: none;
-}
\ No newline at end of file
+}
+
+#info-window-content {
+    padding: 0px;
+    margin: 0px;
+}
```

Comment:
is this even needed if we are doing 0px on both? Are you overriding a default setting?

Reply:
Yes, I am overriding some of the bootstrap styles that come with using the container. I did it on  the infowindow things to use as much room as possible since its pretty small.